<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pod</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tabFind">
        <attribute name="icon">
         <iconset>
          <normaloff>ui/search.png</normaloff>ui/search.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Find podcasts</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Find new podcasts</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Sources:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbxSearchType">
             <property name="toolTip">
              <string>Select podcast sources to search</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="txtSubSearch">
             <property name="placeholderText">
              <string>Search online for podcasts</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSubSearch">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>ui/search.png</normaloff>ui/search.png</iconset>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListView" name="lstSubSearchResults"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Add by RSS URL:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtRssUrl"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnAddRss">
             <property name="text">
              <string>Subscribe</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabSubs">
        <attribute name="icon">
         <iconset>
          <normaloff>ui/antenna.png</normaloff>ui/antenna.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Subscriptions</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <widget class="QLabel" name="lblEps_3">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Subscriptions</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblLabel">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Filter:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtSubFilter">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSortSubs">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>ui/sort-descending.png</normaloff>ui/sort-descending.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnRefreshSubs">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>ui/reload.png</normaloff>ui/reload.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="lstSubs">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>item1</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabEps">
        <attribute name="icon">
         <iconset>
          <normaloff>ui/headphone.png</normaloff>ui/headphone.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Episodes</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QLabel" name="lblEps">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Episodes</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblCurrentShowTitle">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Filter:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtEpFilter">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSortEps">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>ui/sort-descending.png</normaloff>ui/sort-descending.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnRefreshEps">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>ui/reload.png</normaloff>ui/reload.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="lstEps">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="viewMode">
            <enum>QListView::ListMode</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>All Episodes</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="8">
            <widget class="QPushButton" name="btnRefreshEps_2">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>ui/reload.png</normaloff>ui/reload.png</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QLineEdit" name="txtAllEpFilter">
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Filter:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QPushButton" name="btnSortEps_2">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>ui/sort-ascending.png</normaloff>ui/sort-ascending.png</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="lblEps_2">
             <property name="minimumSize">
              <size>
               <width>219</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>All Episodes</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="lstAllEps">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblNowPlaying">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Now playing:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="lblCurrentPodTitle">
           <property name="text">
            <string>Podcast Title</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblCurrentEpTitle">
           <property name="text">
            <string>Episode Title</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="lblDuration">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblDate">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSlider" name="sldProgress">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>10</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblProgress">
         <property name="text">
          <string>Done / Total</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="lytControls">
       <item row="1" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblVolume">
           <property name="text">
            <string/>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="pixmap">
            <pixmap>ui/volume3.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sldVolume">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblVolValue">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>100</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="3">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="btnPrev">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>ui/back.png</normaloff>ui/back.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRwd">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>ui/rewind.png</normaloff>ui/rewind.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPlayPause">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>ui/play.png</normaloff>ui/play.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnFwd">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>ui/forward.png</normaloff>ui/forward.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnNext">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>ui/next.png</normaloff>ui/next.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="7">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="lblPlayerState">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblBuffering">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spcControls">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnSettings">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>ui/settings.png</normaloff>ui/settings.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="nobs.qrc"/>
 </resources>
 <connections/>
</ui>
